src/app/static/src/js/utils/types.ts

```diff
@@ -128,4 +128,17 @@ export interface ChildCustomStyle {
 export interface Error {
     code: string;
     message: string | null;
+}
+
+export interface WorkflowReportWithDependency {
+    name: string,
+    instance?: string
+    params?: Record<string, string>,
+    depends_on?: string[]
+}
+
+export interface Dependency {
+    missing_dependencies: Record<string, string[]>,
+    reports: WorkflowReportWithDependency[],
+    refs: string
```

Comment:
> * Non-default parameters are listed as defaults (there is no default for the nmin parameter in the 'other' report, yet both values are listed as 'default' parameters as well as shown as non-defaults.

This is clearer in my other branch/ticket. However, if you rerun your dev/dependencies you should see report name `default-param` within `other` branch, this will display list of default params.